Mission 3/NMCA_003/NMCA_003_strings.lua

```diff
@@ -402,14 +404,15 @@ ThirdCrystalReclaimed = {
 
 -- Fourth crystal is reclaimed, arty satellite
 FourthCrystalReclaimed = {
-    {text = '[Nichols]: Fourth crystal reclaimed, arty is ready...', vid = 'N03_FourthCrystalReclaimed.sfd', bank = 'N03_VO', cue = 'FourthCrystalReclaimed', faction = 'UEF'},
+    {text = '[Nichols]: Fourth crystal reclaimed, the ballistic computers and target relays of the ship are coming online again. It can now control an orbital support statellite.', vid = 'N03_FourthCrystalReclaimed.sfd', bank = 'N03_VO', cue = 'FourthCrystalReclaimed', faction = 'UEF'},
```

Comment:
when the table is called `FourthCrystalReclaimed` and you add multiple dialogues into it, name the videos and cues with _1 _2 _3
Thats how its done for the rest of the dialogues